### ISSUE TITLE
Design App UI

### ISSUE THREAD

**Issue by Foxworth22** [Bestagons/quack#4]
**Fill in the blanks**:
As a student, I would like to see the app, so that I know how to navigate the app with minimal tutorials.

**Definition of Done - Check as you go!**
- [ ] Tasks have been implemented
- [ ] Documented
- [ ] Passing Tests for each task
- [ ] Pull Request made
- [ ] Code Reviewed

**Comment by RafaelPiloto10** [comment]
I think this is a duplicate of #26. Once #26 is complete, you can make more specific "tickets" for each view/screen rather than 1 large ticket for the entire UI. If you agree, please close this issue
